src/main/kotlin/nl/info/zac/zaak/ZaakService.kt

```diff
@@ -226,6 +231,16 @@ class ZaakService @Inject constructor(
         }
     }
 
+    fun setOntvangstbevestigingVerstuurdIfNotHeropend(zaak: Zaak) {
+        val statusType: StatusType? = zaak.status?.let { statusUuid ->
+            val status = zrcClientService.readStatus(statusUuid)
+            ztcClientService.readStatustype(status.statustype)
+        }
+        if (!StatusTypeUtil.isHeropend(statusType)) {
```

Comment:
Hmm, but `statusType` can be null I read from the code above. Why call this `isHeropend` function if the argument is null? Normally we never want to call functions with null arguments. At very least this is very misleading but generally bad practise anyway. What is the intended behaviour if the status type is null?

Reply:
ah, I see that that `isHeropend` Java method does take a null argument.. argh. too bad. We should refactor this really because null arguments are really bad practise but let's do that in another PR. For now to make it little clearer you could add the `@Nullable` annotation to those methods in `StatusTypeUtil`.